packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -446,6 +377,8 @@ export function TransferPanel() {
     }
   }, [amount, selectedToken, nativeCurrency])
 
+  const { gasSummary, gasSummaryStatus } = useGasSummary()
```

Comment:
NIT: Any reason why gasSummary is initialised so much later in the file? It'd be better if it's present at the top.

Reply:
good point, the original code was around here

i've also moved up `useTransferReadiness` now